main.py

```
@@ -10,6 +10,26 @@
 from google.adk.cli.fast_api import get_fast_api_app
 from datetime import datetime
 from starlette.middleware.base import BaseHTTPMiddleware
+import vertexai
+from vertexai.generative_models import GenerativeModel
```

Ruff (0.12.2)

`vertexai.generative_models.GenerativeModel` imported but unused

Remove unused import: `vertexai.generative_models.GenerativeModel`

(F401)